src/index.ts

```diff
@@ -35,20 +35,21 @@ export const runTopologyWithNats =
     const { unpack, loadSnapshot, persistSnapshot } = fns
     const data = unpack(msg.data)
     const numAttempts = msg.info.redeliveryCount
-    const streamData = getStreamDataFromMsg(msg)
     const isRedelivery = numAttempts > 1
+    debug('Redelivery %s', isRedelivery)
     const { emitter, promise, getSnapshot } = isRedelivery
       ? // Resume topology based on unique stream data
-        resumeTopology(spec, await loadSnapshot(streamData))
+        resumeTopology(spec, await loadSnapshot(msg))
       : // Run topoloty with data from msg
-        runTopology(spec, dag, { data, meta: streamData })
+        runTopology(spec, dag, { data })
```

Comment:
One thing I noticed is that there's no way of passing `{include,exclude}Nodes` to `runTopology`

Reply:
The thinking was that you could just use the exported fn `filterDAG` from `topology-runner`.

Reply:
We should only have one way of doing things. Either `filterDAG` or `include`/`exclude` for both.

Reply:
Makes sense, but I think that can be addressed in a separate PR.

Reply:
I'd likely just remove it from `runTopology` if we want to keep things consistent.

Reply:
Yeah I don't have any strong opinions. Something we should consider is that if we go with `filterDAG`, every job would still need to take `include/exclude` nodes, so we have the ability to publish a message to run a subset of the dag. E.x:

```
./pub.js federalSamOpportunity '({ params: { ... }, includeNodes: [...] })'
```